package.json

```diff
@@ -119,7 +119,7 @@
     "terra-base": "^5.0.0",
     "terra-dev-site": "^6.5.0",
     "terra-disclosure-manager": "^4.9.0",
-    "terra-toolkit": "^5.0.0",
+    "terra-toolkit": "git://github.com/cerner/terra-toolkit.git#upgrade-axe-core",
```

Comment:
Should I revert this or mark it `Do Not Merge` until [#343](https://github.com/cerner/terra-toolkit/issues/343) is fixed and merged. 
cc @benbcai

Reply:
@saket2403 Please revert this. This PR can be merged whenever it is ready and does not depend on #343.

Reply:
Reverted in 1e6f2a7